src/main/java/core/basesyntax/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : data) {
+            if (line.startsWith("type")) {
```

Comment:
Hardcoded strings like "type" should be avoided. Consider defining them as constants, especially if they are used in multiple places, to make the code more maintainable and less error-prone.